File: Java/Human.java
public class Human{
    private String nik;
    private String name;
    private char gender;

    public Human(){
        this.nik = "";
        this.name = "";
        this.gender = '-';
    }

    public Human(String nik,String name, char gender){
        this.nik = nik;
        this.name = name;
        this.gender = gender;
    }

    //Get name
    public String getNik(){
        return this.nik;
    }
    //Set name
    public void setNik(String nik){
        this.nik = nik;
    }

    //Get name
    public String getName(){
        return this.name;
    }
    //Set name
    public void setName(String name){
        this.name = name;
    }
    
    //Get gender
    public char getGender(){
        return this.gender;
    }
    //Set gender
    public void setGender(char gender){
        this.gender = gender;
    }
    
    public void eat(){
        System.out.println(this.name + " is eating!");
    }

    public void sleep(){
        System.out.println(this.name + " is sleeping!");
    }
}

File: C++/Human.cpp
class Human{
    private:
        string nik;
        string nama;
        char gender;
    
    public:

        Human(){

        }

        /* Setter and Getter NIK*/
        void setNik(string nik){
            this->nik = nik;
        }

        string getNIK(){
            return this->nik;
        }

        /* Setter and Getter Nama*/
        void setName(string nama){
            this->nama = nama;
        }

        string getName(){
            return this->nama;
        }

        /* Setter and Getter Gender*/
        void setGender(char gender){
            this->gender = gender;
        }

        char getGender(){
            return this->gender;
        }

        ~Human(){
        }
};